src/utils/toHoursAndMinutes.js

```diff
@@ -0,0 +1,12 @@
+const toHoursAndMinutes = (timeInMinutes) => {
+  const hours = Math.floor(timeInMinutes / 60);
+  const minutes = timeInMinutes % 60;
+
+  if (minutes < 10) {
+    return `${hours}h 0${minutes}min`;
```

Comment:
I'm not sure if a leading 0 is necessary. it seems quite unnatural to see e.g. `1h 06min` 🤔